pkgs/development/libraries/ndn-cpp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, openssl, doxygen, zlib, pkgconfig, protobuf }:
+let
+  version = "v0.15";
```

Comment:
Please bump version to 0.16.

Reply:
I'm actually rewriting it in Rust. I'll close this PR, I won't be maintaining it.